integration/spark-common/src/main/scala/org/apache/carbondata/events/Events.scala

```diff
@@ -198,3 +198,10 @@ trait BuildDataMapEventsInfo {
   val identifier: AbsoluteTableIdentifier
   val dataMapNames: scala.collection.mutable.Seq[String]
 }
+
+/**
+ * EventInfo for IndexServer
+ */
+trait IndexServerEventInfo {
+  val absTable: AbsoluteTableIdentifier
```

Comment:
change `absTable` to `absoluteTableIdentifier`

Reply:
Done